features/entering-rooms.feature

```diff
@@ -7,20 +7,20 @@ Feature: Entering Rooms
   Most frequently, we expect people to enter a Room from their Workspace
   dashboard or by leaving one Room to enter another.
 
-  @built @unimplemented-steps
+  @built
   Scenario: Entering Room via Room Picker from Workspace Dashboard
-    Given I am on a Workspace Dashboard
-    When I tap the Room in the Room Picker
-    Then I am in the Room
+    Given the Workspace Member is on the "System Test" Workspace Dashboard
+    When the Workspace Member taps the "Listed Room 1" Room in the Room Picker
+    Then the Workspace Member is placed in the "Listed Room 1" Room
 
   @built @unimplemented-steps
   Scenario: Entering Room via Room Picker from another Room
     Given I am in a Room
     When I tap a different Room in the Room Picker
     Then I am in the Room
 
-  However, we also want to support Reople entering a Room via
-  the Room's full URI or from a short URI when a Space has a Branded Domain.
+  # However, we also want to support Reople entering a Room via
+  # the Room's full URI or from a short URI when a Space has a Branded Domain.
```

Comment:
Does cucumber-js barf on this? :(. I was really hoping that we could Interleave markdown with the scenarios to make the feature level documentation more communicative.

Reply:
Yes, unfortunately that's the case and I haven't been able to find a way to solve this.
I think moving it up closed to the `Feature:` block would make sense.

Reply:
That makes sense. I’m a bit bummed but I’ll deal :).